### ISSUE TITLE
Load GLTF/GLB misses an object clearly in Blender

### ISSUE THREAD

**Issue by fgadaleta** [bevyengine/bevy#4588]
This is how I load the gltf/glb 

```
struct SceneHandle {
    handle: Handle<Gltf>,
    animations: Vec<Handle<AnimationClip>>,
    instance_id: Option<InstanceId>,
    is_loaded: bool,
    has_camera: bool,
    has_light: bool,
}


fn setup(mut commands: Commands, asset_server: Res<AssetServer>) {
    // Load scene
    let scene_path = "../assets/hangar.gltf".to_string();
    info!("Loading scene {}", scene_path);
    let gltf_scene = asset_server.load(&scene_path);

    commands
    .insert_resource(SceneHandle {
        handle: gltf_scene,
        animations: Vec::new(),
        instance_id: None,
        is_loaded: false,
        has_camera: false,
        has_light: false,
    });
}

```

Blender version
![blender](https://user-images.githubusercontent.com/11614447/165103198-06027bbb-6e4f-4efd-b623-aeff8266c431.png)

How Bevy renders the scene (ladder not present)
![bevy](https://user-images.githubusercontent.com/11614447/165103277-fc667e17-55a0-4651-a7af-f48e6f4120f2.png)

The ladder is actually outside the hangar 
![image](https://user-images.githubusercontent.com/11614447/165103366-7bf405ce-8cb2-4abe-aa65-35af8f68c4c1.png)

**Comment by mockersf** [comment]
could you share your model?

**Comment by fgadaleta** [comment]
the gltf is something like 0.5GB
sure i can share. How?

**Comment by alice-i-cecile** [comment]
Perhaps post a link to Google Drive / Dropbox, or some other file hosting service?

**Comment by fgadaleta** [comment]
Ok I have simplified the .blend file and exported to a smaller gltf that still reproduce the loading error.
Please find the gltf here
https://drive.google.com/file/d/1ePfyheHnzydXRJcSLFmJF9MN1pKea6VJ/view?usp=sharing

**Comment by fgadaleta** [comment]
exactly

**Comment by mockersf** [comment]
This would probably be fixed by #4379, or one of the other issues about transform order.

I tried changing https://github.com/bevyengine/bevy/blob/4bcb3100083fc6e1d4443a97c39c3836563f714e/crates/bevy_transform/src/components/global_transform.rs#L209-L218 to
```rust
    pub fn mul_transform(&self, transform: Transform) -> Self {
        let (scale, rotation, translation) = (Affine3A::from_scale_rotation_translation(
            self.scale,
            self.rotation,
            self.translation,
        ) * Affine3A::from_scale_rotation_translation(
            transform.scale,
            transform.rotation,
            transform.translation,
        ))
        .to_scale_rotation_translation();
        Self {
            translation,
            rotation,
            scale,
        }
    }
```
and it fixes the issue

**Comment by oddfacade** [comment]
yeah, it looks like the current implementation is effectively doing

<img src="https://render.githubusercontent.com/render/math?math=A \cdot B = R_{A}\cdot R_{B}\cdot S_{A}\cdot S_{B}">

when what you actually want is

<img src="https://render.githubusercontent.com/render/math?math=A \cdot B = R_{A}\cdot S_{A}\cdot R_{B}\cdot S_{B}.">

(non-uniform) scaling and rotation don't commute.

**Comment by superdump** [comment]
I was just pointed to this issue. I'm glad it would be fixed by #4379 . Also, the images at the beginning feel like a meme, and I love it. :)

**Comment by mockersf** [comment]
this is fixed now that #4379 has been merged
<img width="1392" alt="Screenshot 2022-07-16 at 03 35 41" src="https://user-images.githubusercontent.com/8672791/179329712-ce2c8965-3ba0-47e7-9032-ecd3dff598fd.png">
(it's now not displaying anything due to #5334 which will be fixed soon)

**Comment by mockersf** [comment]
completely fixed with #5335!
